api/integrations/github/views.py

```diff
@@ -0,0 +1,123 @@
+import requests
```

Comment:
We will need tests for this module.
PS: we will need tests for all the new code introduced in this PR

Reply:
Tests added